### PR TITLE
Adds PayID-Version header to PayIDClient

### PR DESCRIPTION
**Pull request by tedkalaw**
## High Level Overview of Change
This hardcodes the `PayID-Version` header to be `1.0`, per the breaking changes in https://docs.google.com/document/d/1EnOJfpxr4PyM1dMKfYpTeCph2om3F8vZYc4S_PqjNSI/edit#. As is, `xpring-js` does not work for `PayID` `1.0.0` (because it doesn't pass `PayID-Version`) and it does not work for `PayID` `0.1.0` (because it expects to get back an array of `addresses`).

I can update this to take a version if we want but I wanted to go for the smallest change so we can cut a release.

### Context of Change
https://docs.google.com/document/d/1EnOJfpxr4PyM1dMKfYpTeCph2om3F8vZYc4S_PqjNSI/edit#

https://github.com/xpring-eng/Xpring-JS/pull/474/ updates `PayIDClient` to handle the new payload. We still need to support versioning

### Type of Change

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [ ] Tests (You added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation Updates
- [ ] Release

## Before / After
`PayIDClient.addressForPayID` works against `PayID 1.0.0`

## Test Plan
CI

### CODE REVIEW

**Comment by tedkalaw** [comment]
check out the circle CI runs for before and after. that wasn't on purpose, but it shows the change. i'm guessing all xpring-js builds will fail for now

File: src/PayID/pay-id-client.ts [hunk 1]
```diff
@@ -306,6 +306,7 @@ export default class PayIDClient {
   private static makeOptionsWithAcceptTypes(acceptType: string): object {
     const headers = {
       Accept: acceptType,
+      'PayID-Version': '1.0',
```
Comment (keefertaylor) [inline src/PayID/pay-id-client.ts:309]: Does the Key need to be quoted? 

Can you refactor 1.0 to a top level variable?

Comment (hbergren) [reply src/PayID/pay-id-client.ts:309]: > Can you refactor 1.0 to a top level variable?

Why would that be better?

Comment (tedkalaw) [reply src/PayID/pay-id-client.ts:309]: yeah, keys of js objects that have `-` need to be in quotes. if they have `_`, on the other hand, they don't

i'll do the top level var

**Pull request merged**